src/python/pants/jvm/resolve/coursier_setup.py

```diff
@@ -75,6 +75,20 @@
 )
 
 
+POST_PROCESS_COURSIER_STDERR_SCRIPT = textwrap.dedent(
+    """\
+#!{python_path}
+import sys
+from subprocess import run, PIPE
```

Comment:
I'm so happy we chose for `PythonBinary` to be 3.6+

Reply:
BACK IN MY DAY WE HAD TO WALK 15 MILES THROUGH SNOW JUST TO `import Popen`